types/reactstrap/reactstrap-tests.tsx

```diff
@@ -4051,3 +4052,181 @@ function GenericPropExample() {
     </React.Fragment >
   );
 }
+
```

Comment:
I would also recommend adding `CustomInput` to the `GenericPropExample` and `AnyPropExample`.

Reply:
Included in 405abf2